Analysis/PIDTools/src/Compute_dEdxProcessor.cc

```diff
@@ -250,7 +492,7 @@ float Compute_dEdxProcessor::getNormalization(double dedx, float hit, double trk
 }
 
 //create smearing factor
-float Compute_dEdxProcessor::getSmearing(float dEdx){
+double Compute_dEdxProcessor::getSmearing(double dEdx){
   double z=sqrt( -2.0*std::log(dist(*engine)) ) * std::sin( 2.0*3.141592*dist(*engine) );
```

Comment:
Can you also turn this ugly 3.141592 into M_PI from the `cmath` header ?

Reply:
done